.ci/functional_tests_utils.sh

```diff
@@ -102,3 +105,75 @@ function archiveArtifacts() {
   cp ./events_report.txt ./rhche/${JOB_NAME}/${BUILD_NUMBER}/
   rsync --password-file=./artifacts.key -PHva --relative ./rhche/${JOB_NAME}/${BUILD_NUMBER} devtools@artifacts.ci.centos.org::devtools/
 }
+
+function getVersionFromPom() {
+  version=$(mvn org.apache.maven.plugins:maven-help-plugin:evaluate -q -Dexpression=project.parent.version -DforceStdout)
+  echo $version
+}
+
+function getActiveToken() {
+  rm -rf cookie-file loginfile.html
+  if [[ "$USERNAME" == *"preview"* ]]; then
```

Comment:
What about just setting `$preview` value in this if, then both branches of this if, could be squashed into one:
```
    response=$(curl -s -g -X GET --header 'Accept: application/json' "https://api.${preview}openshift.io/api/users?filter[username]=$USERNAME")
    data=$(echo "$response" | jq .data)
    if [ "$data" == "[]" ]; then
      exit 1
    fi     
```